src/ui/components/DataTable/index.tsx

```diff
@@ -382,59 +454,25 @@ const DataTable = ({
 	}
 
 	const colOrderFiltered = state.colOrder.filter(
-		({ hidden, colIndex }) => !hidden && cols[colIndex],
+		({ hidden, colIndex }) => !hidden && state.cols[colIndex],
 	);
 
 	return (
 		<>
-			<CustomizeColumns
-				cols={cols}
-				colOrder={state.colOrder}
-				hasSuperCols={!!superCols}
```

Comment:
How do you deal with superCols (tables with 2 rows of column headers)?

Reply:
currently either you can have super columns on a table or you can have customization, which is similar to the existing system where you can't reorder tables with super cols.
I do think that this should be better addressed. @dumbmatter how do you think they should be handled?

Reply:
That's fine for now, don't want to add more stuff to this PR :)

Eventually, I think each column should somehow be aware of its "parent column", so it can be smart enough to only display the "parent columns" that are necessary. Rather than the current system of there being two separate `cols` and `superCols` arrays.